during_the_workshop.md

```diff
@@ -75,34 +75,99 @@ Now we know how to feed data into Chimera we need to work out how to do this mor
+* You might be wondering how to check the output of your cron jobs. After it runs, and you next interact with the terminal, you will see a message in the terminal saying "You have new mail". You can read the file it tells you about with `cat` or `tail`. E.g. `cat /var/spool/mail/ec2-user`.
 
 We've got some requirements from the CEO:
-* The "last hour" dataset should be regenerated every five minutes.
-* The dataset should include the date and time it was generated in its name. Hint: there is a useful option listed in the cliapp documentation.
-* Opening the site at `/latest` should always display the latest hourly dataset. 
-* All hourly datasets for the last 24 hours should also be kept accessible.
-* Any hourly datasets older than 24 hours should be automatically deleted.
+* A dataset should be generated every five minutes, containing earthquakes in the last hour. **Hint:** there is an option listed in the cliapp documentation for the dataset name.
+* The dataset should include the date and time it was generated in its name. **Hint:** Use the `date` command
+* There should also be a dataset called "latest" which contains the latest data. So opening the site at `/latest` should always display the latest hourly dataset. 
+* Any datasets older than 24 hours should be automatically deleted. More recent ones should be kept accessible. **Hint:** Use the `find` command on the folder containing the datasets. It has options to filter by date/time last modified, and an option to delete the files it finds
 
-## Part 2 (Afternoon)
+## Part 2
 
 ### Creating a local environment
 
 Now you've got the CEO happy it's time to start creating a local development environment. We've started you off by creating a skeleton [Vagrantfile](./Vagrantfile) with some hints as to what different steps there are.
 
-You'll want to copy `webapp` and `cliapp` from your remote VM to use in your local environment.
+Run `vagrant up` in a directory containing a file called `Vagrantfile` and it will create a VM based on that file's instructions. By default, any files in the current directory will also sync the current folder with a folder at `/vagrant` inside the VM.
+
+So to give the Vagrant VM a copy of the `webapp` and `cliapp` executables, you need to copy them from your remote VM into the same folder as your Vagrantfile.
 
 Hint: `scp` might be a helpful program for copying files from a remote machine.
+  
+Your Vagrantfile will need to:
+
+* Set up [port forwarding](https://www.vagrantup.com/docs/networking/forwarded_ports). This means requests from your browser will get forwarded to the VM and you can actually visit the web app.
+* Set up a [provisioning script](https://www.vagrantup.com/docs/provisioning/shell). This script should prepare the environment and install any required tools
+* Use a [trigger](https://www.vagrantup.com/docs/triggers) to start up the webapp after you run "up". (You can just run the `webapp` executable directly).
```

Comment:
I think we should talk about some of the standard vagrant commands here, including:

- vagrant ssh (would argue this is the most important)
- vagrant destroy
- vagrant provision (less important)